code/game/turfs/closed/walls.dm

```diff
@@ -163,12 +163,18 @@
   *When a hulk manages to break a wall using their hulk smash, this deals back damage to the arm used.
   *This is in its own proc just to be easily overridden by other wall types. Default allows for three
   *smashed walls per arm. Also, we use CANT_WOUND here because wounds are random. Wounds are applied
-  *by hulk code based on arm damage and checked after an attack completes.
+  *by hulk code based on arm damage and checked when we call break_an_arm().
   *Arguments:
   **arg1 is the arm to deal damage to.
+  **arg2 is the hulk
  */
-/turf/closed/wall/proc/hulk_recoil(obj/item/bodypart/arm)
+/turf/closed/wall/proc/hulk_recoil(obj/item/bodypart/arm, mob/living/carbon/human/hulkman)
 	arm.receive_damage(brute = 20, blocked = 0, wound_bonus = CANT_WOUND)
```

Comment:
if you add something like

```DM
/turf/closed/wall/proc/hulk_recoil(obj/item/bodypart/arm, mob/living/carbon/human/hulkman, inherit_damage = TRUE)
    if(inherit_damage)
	    arm.receive_damage(brute = 20, blocked = 0, wound_bonus = CANT_WOUND)
```
You can avoid the copypasta, use inherit, and just set `inherit_damage` to FALSE

Reply:
Done, used `var/damage = 20` so that I could set it to both 41 and 0 depending on the wall type.